contributors.yml

```diff
@@ -159,3 +159,8 @@
 - yomeshgupta
 - zachdtaylor
 - zainfathoni
+- dima-takoy
+- anishpras
+- kumard3
+- mattmazzola
+- gmaliar
```

Comment:
Should all of these people be added? Seems like you should only add your own username.

Reply:
You are correct, my bad!